deploy/011-deploy-markets.ts

```diff
@@ -0,0 +1,92 @@
+import { BigNumber, BigNumberish } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { DeployFunction, DeployResult } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { InterestRateModels, getConfig, getTokenConfig } from "../helpers/deploymentConfig";
+
+const mantissaToBps = (num: BigNumberish) => {
+  return BigNumber.from(num).div(parseUnits("1", 14)).toString();
+};
+
+const VTOKEN_DECIMALS = 8;
+const EMPTY_BYTES_ARRAY = "0x";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+  const { tokensConfig, marketsConfig, preconfiguredAddresses } = await getConfig(hre.network.name);
+
+  const comptrollerDeployment = await deployments.get("Unitroller");
+
+  console.log(`Got deployment of Unitroller with address: ${comptrollerDeployment.address}`);
+
+  for (const market of marketsConfig) {
+    const { name, asset, symbol, rateModel, baseRatePerYear, multiplierPerYear, jumpMultiplierPerYear, kink_ } = market;
+
+    const token = getTokenConfig(asset, tokensConfig);
+    let tokenContract;
+    if (token.isMock) {
```

Comment:
Should we also check that the network isn't a test net?

Reply:
I was thinking as an added check because we shouldn't be deploying mocks / underlying tokens on mainnets